homeassistant/components/rflink.py

```diff
@@ -10,15 +10,15 @@
 import logging
 
 import async_timeout
-import voluptuous as vol
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) this position is ok.